trajectorytools/fish_bouts/fish_bouts.py

```diff
@@ -22,16 +45,28 @@ def find_bouts_individual(
                 the next bout
     """
 
-    if find_max_dict is None:
-        find_max_dict = {}
-    if find_min_dict is None:
-        find_min_dict = {}
-
     number_of_frames = speed.shape[0]
 
     # Find local minima and maxima
-    min_frames_ = scipy.signal.find_peaks(-speed, **find_min_dict)[0]
-    max_frames_ = scipy.signal.find_peaks(speed, **find_max_dict)[0]
+    if (find_max_dict is None) and (find_min_dict is None):
+        find_min_dict = {}  # default to  finding minima with scipy
+
+    if find_max_dict is None:
+        min_frames_, max_frames_ = _find_interlaced_peaks(
+            signal=-speed, find_dict=find_min_dict
+        )
+    elif find_min_dict is None:
+        max_frames_, min_frames_ = _find_interlaced_peaks(
+            signal=speed, find_dict=find_max_dict
+        )
+    else:
+        warnings.warn(
+            "Using both find_min_dict and find_max_dict is deprecated. "
+            "In future, specify only one such argument.",
```

Comment:
I am not satisfied with the Deprecation Warning text, but I'm not sure what is best to say here.

Reply:
We can say "is discouraged as it produces inconsistent results"? Or if we do not explain what inconsistent means it is a bit of a mistery?

Reply:
I opted for a vague but satisfying explanation,
```
"...it can miss maxima and minima of bouts, yielding erroneous results."
```